src/config.ts

```diff
@@ -40,6 +41,7 @@ const config: BotConfig = {
   environment: process.env.NODE_ENV as Environment,
   guild_id: process.env.GUILD_ID,
   imp_spotting_time: parseInt(process.env.IMP_SPOTTING_TIME ?? '1'),
+  statistics_id: process.env.STATISTICS_ID,
```

Comment:
Similar to ranks, can you put this under a `channels` object:

```suggestion
  channels: {
    statistics: process.env.STATISTICS,
  }
```

I should probably done this with the channel permissions within commands too.